_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -156,6 +205,23 @@ class AnnualIncomeSection extends React.Component {
               value={this.props.data.childrenOtherIncome}
               onValueChange={(update) => {this.props.onStateChange('childrenOtherIncome', update);}}/>
         </div>
+      </div>);
+    }
+
+    if (this.props.reviewSection) {
```

Comment:
Instead of having a separate prop here, could you just use the `sectionComplete` one? And move this `editButton` into `content` if `sectionComplete` is true?